src/pages/academy/blog/posts/preview/index.tsx

```diff
@@ -38,7 +35,8 @@ import { cms_assets_end_point } from 'common/constants'
 import RightArrow from 'images/svg/tools/black-right-arrow.svg'
 
 const BlogPreview = () => {
-    const [is_mobile] = useBrowserResize(992)
+    const [is_mobile] = useBrowserResize(475)
+    const [is_little_mobile] = useBrowserResize(400)
```

Comment:
I know this was done in the original code in `article.tsx`. However, the way this hook was setup was not intended for multiple use case in the same file. If you need multiple size checks, you can use the new `useWindowSize` hook instead and check for `< 475` and `< 400`.

Reply:
Resolved. Thanks for explain about new hook!